api/src/main/java/org/ehcache/spi/service/PluralService.java

```diff
@@ -22,9 +22,12 @@
 import java.lang.annotation.Target;
 
 /**
- * @author Alex Snaps
+ * Used to indicate that a {@link Service} subtype is permitted to have more than
+ * one concrete implementation registered with a {@link org.ehcache.spi.ServiceProvider}.
+ *
+ * @author Clifford W. Johnson
```

Comment:
@ljacomet you just purged all the author tags right?  Do we care if they start coming back?